acceptance-tests/src/test/java/tech/pegasys/eth2signer/dsl/signer/Signer.java

```diff
@@ -30,31 +32,32 @@
 public class Signer {
 
   private static final Logger LOG = LogManager.getLogger();
-  private static final String PROCESS_NAME = "Eth2Signer";
 
-  private final Eth2SignerProcessRunner runner;
+  private final Eth2SignerRunner runner;
   private final String hostname;
   private final String urlFormatting;
   private final Vertx vertx;
   private HttpClient httpClient;
 
   public Signer(final SignerConfiguration signerConfig) {
-    this.runner = new Eth2SignerProcessRunner(signerConfig);
+    this.runner = Eth2SignerRunner.createRunner(signerConfig);
     this.hostname = signerConfig.hostname();
     urlFormatting = "http://%s:%s";
```

Comment:
nit: the url formatting can be a constant.

Reply:
done